tools/ui-components/src/form-group/form-group.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { FormControl } from '../form-control/.';
+import { FormGroup } from '.';
+
+describe('<FormGroup />', () => {
```

Comment:
Here is the list of tests that we would need for this component.
https://github.com/react-bootstrap/react-bootstrap/blob/master/test/FormGroupSpec.tsx

Reply:
added the input and the check for children test, `div` test is actually counter-intuitive, because the `FormGroup` should have an `as` props that changes the type of the component, aka if we need `fieldset` for accessibility reasons, we shouldn't be blocked by test.